docs/DesignDocs/AttestedTLSAPIs.md

```diff
@@ -0,0 +1,1004 @@
+Attested TLS APIs
+==========================
+
+Motivation
+------------
+
+An important use case for an enclave is establishing an attested
+TLS channel. The attested TLS channel allows a remote party
+to authenticate the enclave identity during a TLS handshake,
+which bridges the data-in-use and data-in-transit protection.
+The Open Enclave SDK demonstrates this use case in the
+[attested TLS samples](/samples/attested_tls) with the use of two
+APIs, `oe_get_attestation_certificate_with_evidence_v2` and
+`oe_verify_attestation_certificate_with_evidence_v2`.
+The `oe_get_attestation_certificate_with_evidence_v2` API allows
+a developer to obtain a self-signed certificate with the enclave
+evidence embedded, which then is used during the TLS handshake.
+More importantly, the API binds the public key of the certificate
+with evidence such that the TLS channel based on the public key
+can be *attested*. To validate the certificate, the receiver uses the
+`oe_verify_attestation_certificate_with_evidence_v2` API, which
+assesses the trustworthiness of the evidence (e.g., based on the DCAP client for SGX).
+In addition, the API outputs a list of claims, allowing the developer
```

Comment:
Since the receiver is a relying party, not a verifier, clarify that the list of claims returned is
NOT the list of claims in the evidence, it's a list of claims from the attestation result returned by the verifier.